source/quick-start.txt

```diff
@@ -16,6 +16,7 @@
 
 .. meta::
    :keywords: get started, tutorial, code example
+   :description: Configure the MongoDB Kafka Connector to transfer data between MongoDB and Apache Kafka using a sandbox environment.
```

Comment:
the product name is used inconsistently in the descriptions. Sometimes we use MongoDB Kafka Connector, sometimes we use Kafka Connector for MongoDB. Can we make this consistent? I think it should be MongoDB Kafka Connector.